Docs/source/install/hpc/cori.rst

```diff
@@ -297,8 +297,13 @@ regime), the following set of parameters provided good performance:
 
 * **4 MPI ranks per Haswell node** (2 MPI ranks per `Intel Xeon E5-2698 v3 <https://ark.intel.com/content/www/us/en/ark/products/81060/intel-xeon-processor-e5-2698-v3-40m-cache-2-30-ghz.html>`_), with ``OMP_NUM_THREADS=16`` (which uses `2x hyperthreading <https://docs.nersc.gov/jobs/affinity/>`_)
 
-GPU
-^^^
+GPU (V100)
+^^^^^^^^^^
 
 Due to the limited amount of GPU development nodes, just request a single node with the above defined ``getNode`` function.
 For single-node runs, try to run one grid per GPU.
+
+A multi-node batch script template can be found below:
```

Comment:
Could you add a line that reminds the user to either source the `gpu_warpx.profile`, or do `module load cgpu` before submitting the job?

Reply:
Will do, but there is a big red block already :)

![Screenshot from 2021-09-22 11-40-15](https://user-images.githubusercontent.com/1353258/134403758-648822ab-7787-4fa8-9122-2ed24545369e.png)

Reply:
Done